scripts/generate-icons.js

```diff
@@ -73,7 +75,7 @@ const generateSvgImports = svg => {
 
 const generateIconList = svg => {
   return Object.keys(svg).map(key => {
-    if (key.includes('-')) {
+    if (nameDelimiter.some(i => key.includes(i))) {
```

Comment:
Would we be able to reduce duplication of the list of character replacements if we ran `camelize` all the time without checking for a list of characters?

Or, could the logic be grouped together, for example, putting the list of characters and conditional checks inside `camelize`?

Right now, updates require changes in the `camelize` function as well as the `nameDelimiter` array. These two are defined quite far apart in the code and then used in a third place: lines 15, 51, and 78.

Reply:
Yeah grouping the logic makes sense to me